client/my-sites/plugins-wpcom/info-header.jsx

```diff
@@ -1,23 +1,19 @@
 import React from 'react';
 
-import Card from 'components/card';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
 
 export const InfoHeader = React.createClass( {
 	render() {
 		return (
-			<Card className="wpcom-plugins__info-header">
-				<div>
-					Uploading and installing your own plugins is not
-					available on WordPress.com, but we offer the most
-					essential ones below.
-					<a
-						href="https://en.support.wordpress.com/plugins/"
-						target="_blank"
-					>
-						Learn more
-					</a>.
-				</div>
-			</Card>
+			<Notice
+				showDismiss={ false }
+				text={ "Uploading and installing your own plugins is not available on WordPress.com, but we offer the most essential ones below." }
+			>
+				<NoticeAction href="https://en.support.wordpress.com/plugins/">
+					{ "Learn More" }
+				</NoticeAction>
```

Comment:
@drw158 this opens up in the current browser window and I was wondering if it should open in a new tab